src/imitation/scripts/config/train_adversarial.py

```diff
@@ -88,22 +91,23 @@ def plots():
   plot_interval = 10
 
 
-# Standard Gym env configs
-
 @train_ex.named_config
 def acrobot():
   env_name = "Acrobot-v1"
+  rollout_hint = "acrobot"
```

Comment:
Why not just store rollouts under `env_name` rather than the named config name / `rollout_hint`? This seems like it'd allow us to get rid of a lot of code.

Reply:
If I don't do this, then the bash scripts need to contain a mapping from `env_config_name` => `env_name`.